<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaPresidente</class>
 <widget class="QMainWindow" name="TelaPresidente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Presidente</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>700</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Presidente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="esquerda">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>560</y>
      <width>20</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="direita">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>560</y>
      <width>20</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
     </font>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pass">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>590</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Passar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>560</y>
      <width>50</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jogar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>450</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Sua vez</string>
    </property>
   </widget>
   <widget class="QPushButton" name="novo_jogo">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>240</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Novo Jogo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="instructions">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>240</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Instructions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="voltar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
     </font>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Digite seu nome:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>200</y>
      <width>115</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jogador</string>
    </property>
   </widget>
   <widget class="QLabel" name="west">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>50</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
     </font>
    </property>
    <property name="text">
     <string>West</string>
    </property>
   </widget>
   <widget class="QLabel" name="nome_jogador">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>475</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="east">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>50</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
     </font>
    </property>
    <property name="text">
     <string>East</string>
    </property>
   </widget>
   <widget class="QLabel" name="north">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>10</y>
      <width>50</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
     </font>
    </property>
    <property name="text">
     <string>North</string>
    </property>
   </widget>
   <widget class="QLabel" name="text">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>700</width>
      <height>500</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
